src/runtime_src/xdp/profile/device/device_intf.cpp

```diff
@@ -706,6 +707,7 @@ DeviceIntf::~DeviceIntf()
             //  }
             //  break;
             //}
+            // case ACCEL_DEADLOCK_DETECTOR: break;
```

Comment:
Similar question to the above one.

Nutshell: If the value isn't being used, it should not be commented out.